sdk/src/main/java/com/ingresse/sdk/services/UserService.kt

```diff
@@ -225,46 +237,51 @@ class UserService(private val client: IngresseClient) {
     }
 
     /**
-     * Get user transfers data
+     * Event attributes
      *
      * @param concurrent - parameters to concurrent request
      * @param request - parameters required to request
      * @param onSuccess - success callback
      * @param onError - error callback
      * @param onConnectionError - connection error callback
      */
-    fun getUserTransfersData(concurrent: Boolean = false,
-                             request: UserTransfersData,
-                             onSuccess: (Array<UserTransfersJSON>) -> Unit,
-                             onError: (APIError) -> Unit,
-                             onConnectionError: (error: Throwable) -> Unit) {
+    fun getEventAttributes(concurrent: Boolean = false,
+                           request: EventAttributes,
+                           onSuccess: (EventAttributesJSON) -> Unit,
+                           onError: (APIError) -> Unit,
+                           onConnectionError: (error: Throwable) -> Unit) {
 
-        var call = service.getUserTransfers(
-            userId = request.userId,
+        val filters = listOf("accepted_apps", "advertisement", "custom_code",
```

Comment:
I think it's better use or an enum or constant val for this.

Reply:
I saw that it is not necessary to pass all the filters in the parameter, if empty the service already return all the filters https://github.com/ingresse/api/blob/master/docs/public-group-event.apib#L142